src/test/unittest/ut_fh.c

```diff
@@ -158,8 +162,16 @@ ut_fh_open_handle(const char *file, int line, const char *func,
 		dwDesiredAccess = GENERIC_READ;
 	else if (acc == FH_WRONLY)
 		dwDesiredAccess = GENERIC_WRITE;
+	else if (acc == FH_EXEC)
+		dwDesiredAccess = GENERIC_EXECUTE;
 	else if (acc == FH_RDWR)
 		dwDesiredAccess = GENERIC_READ | GENERIC_WRITE;
+	else if (acc == FH_EXEC_READ)
+		dwDesiredAccess = GENERIC_EXECUTE | GENERIC_READ;
+	else if (acc == FH_EXEC_WRITE)
+		dwDesiredAccess = GENERIC_EXECUTE | GENERIC_WRITE;
+	else if (acc == FH_ACCMODE)
+		dwDesiredAccess = GENERIC_ALL;
```

Comment:
Since you changed ut_fh access flags to be a bitmask and dwDesiredAccess is a bitmask you can translate them in a more straightforward way.